src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto&display=swap");
+
+@media (max-width: 1024px) {
+  .header {
+    max-width: 1024px;
+  }
```

Comment:
Remove the 'max-width' rule for the .header class inside the media query, as it's not necessary for the given task.